.github/workflows/node.js.yml

```diff
@@ -10,8 +10,30 @@ on:
         branches: [master]
 
 jobs:
+    vector_skin:
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: "Clone vector skin repository"
+              run: |
+                  git clone "https://gerrit.wikimedia.org/r/mediawiki/skins/Vector" skins/Vector
+            - name: "Add vector skin config to LocalSettings.php file"
```

Comment:
Add this after setting up mediawiki-docker

Reply:
@und3fined-v01d, I still don't quite get why we need to add this after setting up mediawiki-docker. I thought by the time MediaWiki-docker runs the skins settings and configurations should be available for use. More insight on this would be great. Thanks

Reply:
@hariclerry 

Just because something runs doesnt necessitate its the right way to do things. We add the vector skins later because that is how it should be chronologically

To give a simple analogy: we set up a mobile phone first and then install apps

Similarly, we should setup mediawiki(the phone) first before adding the plugin to LocalSettings.php
This is what the DEVELOPERS.md file says too

Reply:
Make sense. Thanks